com.unity.multiplayer.mlapi/Tests/Editor/Placeholder.cs

```diff
@@ -0,0 +1,17 @@
+using NUnit.Framework;
+
+namespace MLAPI.Tests
+{
+    public class PlaceholderTests
```

Comment:
Did you mean to leave this in?  Seems like either we would want to remove it, add some comments on why it's there or actually implement some tests

Reply:
yes, that was intentional. I commented a note inline indicating that.